weather_test.go

```diff
@@ -35,7 +36,12 @@ func TestGetSendsCorrectURL(t *testing.T) {
 	t.Parallel()
 	var w weather.Conditions
 	s := httptest.NewTLSServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		fmt.Println("Inside the handler func")
+		data, err := ioutil.ReadFile("testdata/weather.json")
+		if err != nil {
+			fmt.Println("Error: ", err)
+			return
+		}
+		fmt.Fprintln(w, string(data))
```

Comment:
Great! We could also have opened the file using `os.Open`, and then used `io.Copy` to copy the contents of the file (which is an `io.Reader`) to the ResponseWriter (which is an `io.Writer`). But this is totally fine too.